develop/parachains/deployment/coretime-renewal.md

```diff
@@ -0,0 +1,257 @@
+
+For more comprehensive information about the coretime sales process, refer to the [Coretime Sales](https://wiki.polkadot.network/learn/learn-agile-coretime/#coretime-sales){target=\_blank} section in the Polkadot Wiki.
+
+## Renewal Timing
+
+While renewals can technically be made during any phase, it is strongly recommended to complete renewals during the interlude phase. Delaying renewal introduces the risk that the core could be sold to another participant in the market, preventing successful renewal. Renewals must be initiated well in advance to avoid the aforementioned scenario. 
+
+For example, if you purchase a core in bulk sale #1, you obtain coretime for the upcoming bulk period (during which bulk sale #2 takes place).
+Your renewal must be completed during bulk sale #2, ideally during its interlude phase, to secure coretime for the subsequent period.
+
+## Manual Renewal
+
+Cores can be renewed by issuing the [`broker.renew(core)`](https://paritytech.github.io/polkadot-sdk/master/pallet_broker/pallet/struct.Pallet.html#method.renew){target=\_blank} extrinsic during the coretime sale period. While this process is straightforward, it requires manual action that must not be overlooked. Failure to complete this renewal step before all available cores are sold could result in your parachain being unable to secure a core for the next operational period.
+
+To manually renew a core:
+
+1. In [Polkadot.js Apps](https://polkadot.js.org/apps/#/explorer){target=\_blank}, connect to the Coretime chain, navigate to the **Developer** dropdown and select the **Extrinsics** option
+
+    ![](/images/develop/parachains/deployment/coretime-renewal/coretime-renewal-1.webp)
+
+2. Submit the `broker.renew` extrinsic
+    1. Select the **broker** pallet
+    2. Choose the **renew** extrinsic
+    3. Fill in the **core** parameter
+    4. Click the **Submit Transaction** button
+
+    ![](/images/develop/parachains/deployment/coretime-renewal/coretime-renewal-2.webp)
+
+For optimal results, the renewal should be performed during the interlude phase. Upon successful submission, your core will be renewed for the next coretime period, ensuring continued operation of your parachain.
+
+## Auto Renewal
+
+The Coretime auto-renewal feature simplifies the process of maintaining continuous coretime allocation by automatically renewing cores at the beginning of each sale period. This eliminates the need for parachains to manually renew their cores for each bulk period, reducing operational overhead and the risk of missing renewal deadlines.
+
+When auto-renewal is enabled, the system follows this process at the start of each sale:
+
+1. The system scans all registered auto-renewal records
+2. For each record, it attempts to process renewal payments from the task's sovereign account (which is the sibling account on the Coretime chain derived from the Parachain ID)
```

Comment:
Should link to something about sibling account derivation, otherwise it's confusing.

Reply:
Do you have a resource in mind that you'd suggest linking to?